documentation/source/python_entry_point.rst

```diff
@@ -0,0 +1,41 @@
+Python Entry Point
+==================
+
+By default, cocotb starts the :class:`~cocotb.regression.RegressionManager` as the testing framework for cocotb.
+However, there are many reasons why a user might want to replace the whole or a part of the existing Python framework with an alternative.
+To support this, the environment variable :envvar:`PYTHON_ENTRY` was created.
```

Comment:
```suggestion
To support this, the environment variable :envvar:`PYTHON_ENTRY` can be used.
```
The "was created" makes sense now in this PR, but in the future, the variable is just there, and referring to the creation may sound... quirky? (Not a native speaker, so my feeling may be wrong.)